src/helpers.js

```diff
@@ -83,6 +83,9 @@ export function ruleURI(ruleId) {
     case 'react':
       return `https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/${ruleName}.md`
 
+    case 'standard':
+      return `https://github.com/xjamundx/eslint-plugin-standard#rules-explanations`
```

Comment:
You can use single-quotes here, backticks are only allowed by our linting config when using features like string interpolation (e.g. on the line above).